jenkins/pipelines/cd/tiup/tiup-mirror-update-multi-products.groovy

```diff
@@ -274,12 +264,15 @@ def update_ctl = { version, os, arch ->
         """
     }
 
+    // tiup-ctl 一般不会变更，可以固定使用 v1.8.1 版本
```

Comment:
should not hard code it everywhere, it better use a constant declared with `final`